docs/changelog.md

```diff
@@ -57,6 +57,7 @@ title: Changelog
 * [MAINTENANCE] Fix docstring on expectation (#5204) (thanks @andyjessen)
 * [MAINTENANCE] Improve NaN handling in numeric ParameterBuilder implementations (#5226)
 * [MAINTENANCE] Update type hint and docstring linter thresholds (#5228)
+* [FEATURE] Improve Trino support and enable test (#5261)
```

Comment:
Please remove this line. After this PR is approved and merged, the changelog will be automatically updated on the next release.